### PR TITLE
Social links fix

### LINKED ISSUE #1127: [NG] social links not working properly on community members page
**Description**
The[ community members page](https://layer5ng.netlify.app/community/members)  cards contain social links (`github, twitter, ...`) of the members. When clicked on them they are not directing to the correct page instead the individual member page gets open. 

**Expected Behavior**
The social links on clicked should direct to the correct page. This is probably an issue with CSS of the page 

[Optional] **Additional Context**

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by dDab-panda**
**Description**

This PR fixes #1127

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/sections/Members-grid/index.js [hunk 1]
```diff
@@ -36,9 +36,7 @@ const MembersGrid = props => {
                         <Row>
                             {props.data.allMdx.nodes.map(({ id, frontmatter, fields }) => (
                                 <Col xs={12} sm={6} md={6} lg={4} xl={3} key={id}>
-                                    <Link to={fields.slug}>
-                                        <ProfileCard frontmatter={frontmatter} />
-                                    </Link>
```
Comment (kishore007k) [inline src/sections/Members-grid/index.js:41]: Remove the `Link` tag since it is not used anywhere.

**Review by kishore007k — CHANGES_REQUESTED** [review]

File: src/sections/Members-grid/index.js (hunk 1 above)
Comment (dDab-panda) [reply src/sections/Members-grid/index.js:41]: I think, I have removed it already, and added a cardlink prop

Comment (kishore007k) [reply src/sections/Members-grid/index.js:41]: Yeah @dDab-panda, But, Did you removed it from the import?

Comment (dDab-panda) [reply src/sections/Members-grid/index.js:41]: Oh forgot about that, will remove it

**Review by kishore007k — APPROVED** [review]

**Comment by kishore007k** [comment]
Nice work here @dDab-panda, One last thing could you fix the `DCO`

**Comment by dDab-panda** [comment]
> Nice work here @dDab-panda, One last thing could you fix the `DCO`

Done

**Comment by Tanuj22** [comment]
@dDab-panda can you remove the unused `Link` import.

**Review by kishore007k — APPROVED** [review]

**Review by Jashpatel1 — APPROVED** [review]
Looks good to me. Thanks @dDab-panda

**Pull request merged**